src/greatwall/gui.py

```diff
@@ -1002,6 +1006,9 @@ def config_selecting_derivation_widgets(self):
             ):
                 if idx == 0:
                     selection_button = widgets
+                    selection_button.setEnabled(
+                        False if self.greatwall.current_level == 0 else True
```

Comment:
I add code formating to match our current formating.  However, we intend to support one of the code formating tool, like black, to simplifying this process.